cmd/transform.go

```diff
@@ -94,9 +94,6 @@ func transformHandler(cmd *cobra.Command, flags transformFlags) {
 		if cmd.Flags().Changed(planFlag) {
 			logrus.Fatalf("Error while accessing plan file at path %s Error: %q", flags.planfile, err)
 		}
-		if !cmd.Flags().Changed(sourceFlag) {
-			logrus.Fatalf("Invalid usage. Must specify either path to a plan file or path to directory containing source code.")
-		}
```

Comment:
I think it is okay to expect the source directory to be specified and being mandatory.

Reply:
done!